resources/templates/uma.yml.j2

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
This environment file is pretty oudated in terms of versions (etiss, tvm,...)

Let's make sure to update this file to the latest versions after the merge @SamiraAFg